### PR TITLE
Sync kops e2e image tag, and use a stable one for the blocking job

### PR DESCRIPTION
**Pull request by krzyzacy**
https://github.com/kubernetes/test-infra/issues/2236

### CODE REVIEW

**Comment by fejta** [comment]
Do we know whether this works? Can we create a kops canary job?

**Comment by krzyzacy** [comment]
`kops-runner` did not changed, and the rest should be covered by the canary job?

**Comment by krzyzacy** [comment]
I'll make a canary job in a separate PR then.

File: jobs/config.json [hunk 1]
```diff
@@ -405,7 +405,8 @@
   "args": [
     "--cluster=e2e-kops-aws.test-aws.k8s.io",
     "--env-file=platforms/kops_aws.env",
-    "--env-file=jobs/ci-kubernetes-e2e-kops-aws.env"
+    "--env-file=jobs/ci-kubernetes-e2e-kops-aws.env",
+    "--tag=v20170223-43ce8f86"
```
Comment (zmerlynn) [inline jobs/config.json]: I hate JSON's inability to terminate a list on a comma. :)

**Pull request merged**